.github/workflows/release.yml

```diff
@@ -1,18 +1,22 @@
 name: Publish new release
 on:
   workflow_dispatch:
+    inputs:
+      release_type:
+        description: Type of release (see the semantic versioning guides if you need help determining the appropriate choice)
+        type: string
```

Comment:
I think I want `type: choice` here, the default is loading but it's freeform 

<img width="357" alt="Screenshot 2024-01-03 at 14 39 29" src="https://github.com/ioniccommerce/ionic_langchain/assets/12285859/afc5efff-16a0-42a8-aab2-2911201be666">